src/components/FormComponents/FormElement.js

```diff
@@ -42,9 +42,9 @@ const styles = {
   },
 };
 
-const FormElement = styled(({ className, ...rest }) => (
-  <div className={cx('f-FormEl', className)} {...rest} />
-))`
+const FormElement = styled('div').attrs(({ className }) => ({
+  className: cx('f-FormEl', className),
```

Comment:
Are the other props automatically spread?

Reply:
Yup. `attrs` kinda extends it.